zds/featured/models.py

```diff
@@ -7,6 +7,15 @@
 
 
 class FeaturedResource(models.Model):
+    """
+        A Featured Ressource is an element that will be displayed as a front page news.
+        It displays 3 main information:
```

Comment:
Je suis gêné par le "main" car au final le plus important c'est le lien, non ? Voilà ce que je propose :

    """
        A FeaturedResource is a link to a resource that is featured by the Staff
        It displays 3 main informations:
            - A background picture
            - A title
            - The author(s) of the resource
        Currently, the five newer FeaturedResource are displayed on the front page.
    """